Change Led driver PCA9633 to enable the device at init

daniel-nicolai-hansten opened the pull request:
Update of LED Driver for PCA9633 to take the IC out from sleep at driver initialization.
Fixes #37180

Mani-Sadhasivam commented on drivers/led/pca9633.c:
```diff
@@ -184,7 +186,13 @@ static int pca9633_led_init(const struct device *dev)
+				~PCA9633_MODE1_SLEEP)) {
+		LOG_ERR("LED reg update failed");
+		return -EIO;
+	}
```
new line

Mani-Sadhasivam commented on drivers/led/pca9633.c:
```diff
@@ -184,7 +186,13 @@ static int pca9633_led_init(const struct device *dev)
 		LOG_DBG("Failed to get I2C device");
 		return -EINVAL;
 	}
-
```
Add a comment on what this register setting is doing. Like
`Turn Oscillator ON`

Mani-Sadhasivam reviewed (changes requested):
Please add a proper commit subject and description. The `Fixes` tag could be added to the commit description itself.

gmarull commented:
Thanks for this fix, please, format commit message correctly, e.g. `drivers: led: pca9633: exit sleep mode on init` and add a description of the problem (see https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)

daniel-nicolai-hansten commented:
> Thanks for this fix, please, format commit message correctly, e.g. `drivers: led: pca9633: exit sleep mode on init` and add a description of the problem (see https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)

Done

daniel-nicolai-hansten commented:
> 
> 
> > Thanks for this fix, please, format commit message correctly, e.g. `drivers: led: pca9633: exit sleep mode on init` and add a description of the problem (see https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)
> 
> Done

I think the sign off message are correct now. Please rerun the checks

daniel-nicolai-hansten committed:
drivers: led: pca9633: exit sleep mode on init.

Driver does not correctly take ic out from sleep mode on init.
Added logic to update mode register on init to take the leddriver
out from sleep mode.
See datasheet for register details.
Fixes #37180

Signed-off-by: Daniel N. Hansten <dnh2000@gmail.com>

daniel-nicolai-hansten commented:
> 
> 
> > > Thanks for this fix, please, format commit message correctly, e.g. `drivers: led: pca9633: exit sleep mode on init` and add a description of the problem (see https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)
> > 
> > 
> > Done
> 
> I think the sign off message are correct now. Please rerun the checks

I have now fixed the Autor formatting in my git. Please rerun the checks

daniel-nicolai-hansten commented:
> 
> 
> > > > Thanks for this fix, please, format commit message correctly, e.g. `drivers: led: pca9633: exit sleep mode on init` and add a description of the problem (see https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)
> > > 
> > > 
> > > Done
> > 
> > 
> > I think the sign off message are correct now. Please rerun the checks
> 
> I have now fixed the Autor formatting in my git. Please rerun the checks

Added body to commit message and leaving line 2 blank.
